tests/test_files/markdown/lists.md

```diff
@@ -0,0 +1,40 @@
+This is an unordered list with minuses.
+- item 1
+- item 2
+
+
+This is an unordered list with pluses.
++ item 1
++ item 2
+
+
+This is an unordered list with asterisks.
+* item 1
+* item 2
+
+
+This is an unordered list with bullets.
+• item 1
+• item 2
+
+
+This is an ordered list.
+1. item 1
+2. item 2
+6. item 3
+6. item 4
```

Comment:
did you mean for these to both be 6? they aren't in the html above?

Reply:
That's intentional.  Markdown just requires digits to denote an ordered list.  They don't need to be unique or in order.

Reply:
Can you note that somewhere so that someone doesn't come along later and 'fix' it?